modules/dolt/examples_test.go

```diff
@@ -0,0 +1,96 @@
+package dolt_test
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"path/filepath"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/dolt"
+)
+
+func ExampleRunContainer() {
+	// runDoltContainer {
+	ctx := context.Background()
+
+	doltContainer, err := dolt.RunContainer(ctx,
+		testcontainers.WithImage("dolthub/dolt-sql-server:1.32.4"),
+		dolt.WithConfigFile(filepath.Join("testdata", "dolt_1_32_4.cnf")),
+		dolt.WithDatabase("foo"),
+		dolt.WithUsername("root"),
+		dolt.WithPassword("password"),
+		dolt.WithScripts(filepath.Join("testdata", "schema.sql")),
+	)
+	if err != nil {
+		panic(err)
```

Comment:
In #2193 we are replacing the panics with log.Fatal, including some `gocritic` exclusions. Could you please take a look? Remember to run `make lint` in the module?

Reply:
I removed the panics and ran `make lint`
```bash
dustin@Dustins-MacBook-Pro-3 dolt % make lint
go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.55.2
golangci-lint run --out-format=github-actions --path-prefix=. --verbose -c /Users/dustin/src/testcontainers-go/.golangci.yml --fix
INFO [config_reader] Used config file ../../.golangci.yml 
INFO [lintersdb] Active 13 linters: [errcheck errorlint gci gocritic gofumpt gosimple govet ineffassign misspell nonamedreturns staticcheck testifylint unused] 
INFO [loader] Go packages loading at mode 575 (deps|files|name|types_sizes|compiled_files|exports_file|imports) took 6.415493167s 
INFO [runner/filename_unadjuster] Pre-built 0 adjustments in 595.834µs 
INFO [linters_context/goanalysis] analyzers took 47.412515ms with top 10 stages: buildir: 15.056756ms, gocritic: 6.616042ms, gci: 2.306583ms, gofumpt: 2.229125ms, S1038: 2.181292ms, S1039: 1.310084ms, S1030: 1.166125ms, S1019: 1.063625ms, printf: 1.037487ms, fact_deprecated: 774.621µs 
INFO [runner] fixer took 0s with no stages        
INFO [runner] Issues before processing: 10, after processing: 0 
INFO [runner] Processors filtering stat (out/in): autogenerated_exclude: 10/10, identifier_marker: 10/10, exclude-rules: 2/10, nolint: 0/2, filename_unadjuster: 10/10, path_prettifier: 10/10, skip_dirs: 10/10, cgo: 10/10, exclude: 10/10, skip_files: 10/10 
INFO [runner] processing took 442.87µs with stages: nolint: 159.167µs, identifier_marker: 103.875µs, exclude-rules: 74.25µs, path_prettifier: 53.291µs, autogenerated_exclude: 35.208µs, fixer: 6.708µs, skip_dirs: 6.167µs, cgo: 1.083µs, severity-rules: 624ns, filename_unadjuster: 541ns, max_same_issues: 376ns, skip_files: 249ns, uniq_by_line: 209ns, exclude: 209ns, sort_results: 208ns, source_code: 166ns, diff: 126ns, path_shortener: 124ns, max_from_linter: 124ns, max_per_file_from_linter: 83ns, path_prefixer: 82ns 
INFO [runner] linters took 520.344916ms with stages: goanalysis_metalinter: 519.867625ms 
INFO File cache stats: 2 entries of total size 6.8KiB 
INFO Memory: 71 samples, avg is 31.5MB, max is 98.9MB 
INFO Execution took 6.944488958s
```